.pylintrc

```diff
@@ -0,0 +1,301 @@
+Skip to content
```

Comment:
Was this added manually, or autogenerated by some pylint command? just curious

Reply:
Fixed this obviously wrong `.pylintrc` file